doc/config.rst

```diff
@@ -0,0 +1,670 @@
+
+
+Key Name: KolmogorovPSF
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This PSF class is based on David Kirkby's presentation to the DESC Survey Simulations working group on 23 March 2017.
+
+    https://confluence.slac.stanford.edu/pages/viewpage.action?spaceKey=LSSTDESC&title=SSim+2017-03-23
+
+    (you will need a SLAC Confluence account to access that link)
+
+Required keywords to set:
+""""""""""""""""""""""""""
+
+    * ``airmass`` = *float_value* (default =  None)  The aimass in the direction of the pointing.
+    * ``rawSeeing`` = *float_value*  The FWHM seeing at zenith at 500 nm in arc seconds
+    * ``band`` = *str_value* The filter band of the observation.
+
+
+
+Tree Rings
+----------
+
+Tree-rings are a silicon sensor effect induced by internal electric fields in the 3D structure of the silicon CCD.  The fields are created by internal variations in dopant concentration that form while the silicon boule is being grown.  You can find more about *imSim*'s implementation of tree rings in :ref:`the Tree Ring validation section <tree-ring-label>`. This keyword tells imSim where to find the data file which describes the parameters to be used when the effect is turned on. It creates dictionaries that can be used by the LSST sensor description in :ref:`LSST Camera <LSST-Camera-label>` section below.
+
+Key Name: tree_rings
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Required keywords to set:
+""""""""""""""""""""""""""
+
+    * ``file_name`` = *str_value* (default =  None)  A file name that contains the parameters of the tree ring model for each sensor.
+
+
+Optional keywords to set:
+"""""""""""""""""""""""""
+
+    * ``only_dets`` = *List* (default = None) Only read in the models for the listed sensors in order to save time on startup.
+
+
+
+TeeeRingCenter and TreeRingFunction Value Type
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Once the tree ring models have been read in, you can use them in other parts of the YAML file. The model is keyed on a detector name. An example is shown below:
+
+.. code-block:: yaml
+
+    image:
+        type: LSST_Image
+
+        sensor:
+            type: Silicon
+
+            treering_center: { type: TreeRingCenter, det_name: $det_name }
+            treering_func: { type: TreeRingFunc, det_name: $det_name }
+
+
+Checkpointing
+-------------
+
+As imSim runs, if this option is turned on, it will periodically check-point its progress, writing out its interim output as it runs.  Then, on re-running, it will use this output so as to not redo previous calculations.  This has two main use-cases.  The first is the case where you are rerunning several times. This will avoid recreating sensors that have already been simulated. The 2nd main use case is for if a job is stopped before it completes.  This is particularly common when using a batch system with time-limits.  This option allows you to restart your job and pick where you left off.  These keywords tell *imSim* where to find the checkpoint files and how they are named.
```

Comment:
Fixed.  

The 1st use case if it you did a long run and you generated (say) 20 sensors and now you want to generate 20 additional ones that you didn't simulate before.  This allows you to just simulate those and not-redo what you already did without removing them from the list of what you want to simulate.

Reply:
Ah, so that (first) use case won't work with the "fix" I suggested in Issue #382.  But we could add a feature to check for the existence of the output file(s) and if they exist, don't redo them.  (Optionally, and probably not by default.) That would I think enable that use case pretty effectively.  This would be a new option in LSST_CCD.  Something to consider for 2.1 I suppose.